packaging/windows/split.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+const chunkSize = 1024 * 1024 * 1024 // 1GiB chunk size
+
+func main() {
+	if len(os.Args) != 2 {
+		log.Fatal("Split takes only one argument (the file to split)")
+	}
+
+	if err := split(os.Args[1]); err != nil {
+		log.Fatal(err.Error())
+	}
+}
+
+func split(filePath string) error {
+	bundle, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+	defer bundle.Close()
+
+	bundleFileStat, err := bundle.Stat()
+	if err != nil {
+		return err
+	}
+
+	for i := 0; ; i++ {
+		partFileName := fmt.Sprintf("%s.%d", bundleFileStat.Name(), i)
+
+		err := func() error {
+			partFile, err := os.Create(filepath.Join(filepath.Dir(filePath), partFileName))
+			if err != nil {
+				return err
+			}
+			defer partFile.Close()
+			n, err := io.CopyN(partFile, bundle, chunkSize)
+			fmt.Printf("Copied %d bytes from %s to %s\n", n, bundleFileStat.Name(), partFileName)
+			return err
+		}()
+
+		if err != nil {
+			if errors.Is(err, io.EOF) {
+				break
+			}
```

Comment:
Maybe on a single line? `if err != nil && !errors.Is(err, io.EOF) { return err }`

Reply:
ah it's my suggestion which broke your code /o\ I did not even remember it :(